docs/split_keyboards.md

```diff
@@ -35,44 +48,92 @@ split = Split(
+
+#### `split_flip`
+If your split keyboard use the **same pcb for both side**, but vertically flipped, Set this to `True`, and `False` otherwise. `True` will means everything are the same for both side except that `row_pins` are reversed (as you flip one of the pcb verticaly).
+
+You will need to update your `coord_mapping` (if you use one) if you change this value.
+
+#### `split_target_left`
+The `split_target` refer to the side that will send keypress over usb.
 
-If you're using RP2040 based board and want to use other pins that the one with hardware support of RX/TX, you can try
-using PIO implementation of two-way wire protocol. Typical use cases for it are premade boards, made with QMK's
-bitbanging protocols in mind.
+If you use EE HANDS (`split_side = None`) this parameter will be automatically overriden (a side connected to a usb host will act as a `split_target`).
+
+If setting this manually, be sure to reflect the actual usb connection and make sure to use the same value for both sides !
+
+#### `uart_flip`
+If your boards are connected through the **same** pins (like gpio**4** of board A to gpio**4** of board B): use `uart_flip = True`.
+
+If your boards are connected through **different** pins (like gpio**4** of board A to gpio**10** of board B): use `uart_flip = False`.
+
+For example if using the qwiic/stemma connector of the [KB2040](https://www.adafruit.com/product/5302) (supposed to be for I2C) for the uart connection. You will need to use `uart_flip = True` as for i2c sda connect to sda and scl to scl.
+
+#### `use_pio`
+If you're using RP2040 based board and want to use other pins that the one with hardware support of RX/TX, you can try using PIO implementation of two-way wire protocol. Typical use cases for it are premade boards, made with QMK's bitbanging protocols in mind.
 
 In order to enable it, you must:
 
-- install Circuit Python in 7.2+ version
-- add `adafruit_pioasm.mpy` library to lib or root folder of a board
-- pass `use_pio=True` into `Split()` constructor
+- Install Circuit Python in 7.2+ version
+- Add `adafruit_pioasm.mpy` library to lib or root folder of a board
+- Pass `use_pio=True` into `Split()` constructor
+
+The file is available [here](https://github.com/adafruit/Adafruit_CircuitPython_PIOASM/releases). From the `adafruit-circuitpython-pioasm-x.y-mpy-0.x.y.zip`, copy the `lib` folder (or its content) to the root of your circuitpython drives. Make sure to get the zip corresponding to your circuitpython version as the mpy format can evolve!
+
+Continuing our example of the KB2040, one side's UART being fliped will need to use a pin capable of only tx as rx (and vice-versa). So in order to use that qwiic/stemma connector you will also need `use_pio = True`.
```

Comment:
```suggestion
The file is available [here](https://github.com/adafruit/Adafruit_CircuitPython_PIOASM/releases). From the `adafruit-circuitpython-pioasm-x.y-mpy-0.x.y.zip`, copy the `lib` folder (or its content) to the root of your circuitpython drives. Make sure to get the zip corresponding to your circuitpython version as the mpy format can evolve!

Continuing our example of the KB2040, one side's UART being fliped will need to use a pin capable of only tx as rx (and vice-versa). So in order to use that qwiic/stemma connector you will also need `use_pio = True`.
```
* Please don't use "[here]" as a name for a link. Give it the name or description it's pointing at.
* The second paragraph is very confusing again. When did we get from RP2040 to KB2040? What does "use a pin capable of only tx as rx" mean? Why are we talking about qwicc/stemma suddenly? Sorry, but this is all very incoherent.